website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,49 +1,107 @@
 ## Component API
 
-The Table component itself is where most of the options will be applied. However, the child components can also be used if a custom implementation is needed.
+The Table component itself is where most of the options will be applied. However, the API for the child components are also documented here, in case a custom implementation is desired.
 
-- The `Hds::Table::Tr` component is a template-only component. It supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering) but is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). It can contain `Hds::Table::Th` or `Hds::Table::Td` components.
-- The `Hds::Table::Th` component is a template-only component. It supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering) and can contain interactive elements but is not eligible to receive interactions itself. It‘s unlikely you‘ll need to add interactive elements, as sorting is already provided.
-- The `Hds::Table::Td` component is a template-only component. It supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering) and can contain interactive elements (e.g., `<td><a href="user-info.html">User info</a></td>`) but is not eligible to receive interactions itself (e.g., it cannot have an `onClick` event handler attached directly to it).
+### Table
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="<:head>" @type="named block">
-    This is a named block where the content for the table head (`<thead>`) is rendered.
+    This is a named block where the content for the table head (`<thead>`) is rendered. It is unlikely that most consumers will need to use this named block directly for most use cases.
   </C.Property>
   <C.Property @name="<:body>" @type="named block">
     This is a named block where the content for the table body (`<tbody>`) is rendered.
   </C.Property>
   <C.Property @name="model" @type="array">
-    If defined, sets the data source that gets yielded by the `:body` named block.
+    Indicates the data model to be used by the table.
   </C.Property>
   <C.Property @name="columns" @type="array">
-    Use an `array` hash to define your table columns. While `key` and `label` are required, other options include `isSortable`, `align` (for text-alignment), and `width`.
+  Use a `hash` within the array to define each column. `Key` and `label` values are both required. Add `isSortable` if you would like the column to be sortable.
```

Comment:
Technically is not correct that the `key` (lowercase) is required: it is only if one needs a sortable table, but if the table is static using just  the `label` will work.

Reply:
ok, added clarification